lib/phlex/tag.rb

```diff
@@ -45,6 +45,7 @@ def attributes
       attributes = @attributes.dup
       attributes[:class] = classes
       attributes.compact!
+      attributes[:href] = attributes[:href].gsub("javascript:", "") if attributes[:href]
```

Comment:
Let's use a regular expression to only substitute this if it's the start of the string `^\s*(javascript:)+`. Given this expression, we could use `sub` instead of `gsub`. We can also mutate the original string rather than create a new string with `sub!`.

```suggestion
      attributes[:href].sub!(/^\s*(javascript:)+/, "") if attributes[:href]
```